### DISCUSSION TITLE
Unraid: setcap cap_net_raw=pe /path/to/bin/node

### DISCUSSION THREAD

**Discussion by shimi269** [AlexGustafsson/homebridge-wol#257] [General]
### Issue

When I try to use the shutdown command, I get the following error:

Error: Command failed: net rpc shutdown --ipaddress 192.168.1.XXX --user username%password
/bin/sh: 1: net: not found

I have been able to run the command when using the terminal in Unraid itself, so I know that the command can work, it's just in the docker for Homebridge that I get this error, and not sure how to fix it.

### Environment

```Markdown
- os and os version: Unraid
- node version:
- homebridge version: latest
- homebridge-wol version: latest
```

### Logs

```Markdown
When I follow these steps:
1. Make sure `logLevel` is set to `Debug` on all `NetworkDevice` accessories
2. Run `homebridge`
3. Let it run for a couple of minutes
4. Try to turn a device on or off using the plugin
5. Wait one minute
6. Repeat step 4

I get the following log:

Please paste your log here
```
```


### Notes

_No response_

**Comment by AlexGustafsson** [comment]
The docker container for homebridge runs entirely separately from your computer, where you've tested the `net` command. What you have to do is to ensure that the `net` command is available from within the docker container itself. This can be done ephemerally by attaching to the container and installing the command. In order for the command to always be there though, your best bet is to create a custom docker image derived from homebridge in which you make sure to install the net command.

**Reply by AlexGustafsson** [reply]
I _think_ I've seen this issue mentioned in other discussions here, so make sure to check around to see if someone else have come up with a solution.
